File: test/ocamlTests/IntTest/isOddTest.ml
test "isOdd(8)" (fun () -> expect (Int.isOdd 8) |> toEqual Eq.bool false) ; 
test "isOdd(9)" (fun () -> expect (Int.isOdd 9) |> toEqual Eq.bool true) ; 
test "isOdd(0)" (fun () -> expect (Int.isOdd 0) |> toEqual Eq.bool false) ; 

File: test/fsharpTests/IntTest/isOddTest.fs
testCase "isOdd(8)" 
<| fun _ -> 
    let expected = false
    Expect.equal expected (Int.isOdd 8) "error"
testCase "isOdd(9)" 
<| fun _ -> 
    let expected = true
    Expect.equal expected (Int.isOdd 9) "error"
testCase "isOdd(0)" 
<| fun _ -> 
    let expected = false
    Expect.equal expected (Int.isOdd 0) "error"
